lib/hsa/mcwamp_hsa.cpp

```diff
@@ -4970,14 +4969,14 @@ HSACopy::waitComplete() {
     // Wait on completion signal until the async copy is finishedS
     if (DBFLAG(DB_WAIT)) {
         hsa_signal_value_t v = -1000;
-        if (signal.handle) {
-            hsa_signal_load_acquire(signal);
+        if (_signal.handle) {
+            hsa_signal_load_acquire(_signal);
         }
-        DBOUT(DB_WAIT, "  wait for copy op#" << getSeqNum() << " completion with wait flag: " << waitMode << "signal="<< std::hex  << signal.handle << std::dec <<" currentVal=" << v << "...\n");
+        DBOUT(DB_WAIT, "  wait for copy op#" << getSeqNum() << " completion with wait flag: " << waitMode << "signal="<< std::hex  << _signal.handle << std::dec <<" currentVal=" << v << "...\n");
     }
 
     // Wait on completion signal until the async copy is finished
-    hsa_signal_wait_acquire(signal, HSA_SIGNAL_CONDITION_LT, 1, UINT64_MAX, waitMode);
+    hsa_signal_wait_acquire(_signal, HSA_SIGNAL_CONDITION_LT, 1, UINT64_MAX, waitMode);
```

Comment:
Unless I am mistaken, this is deprecated, with `hsa_signal_wait_scacquire` being the current / preferred form.